src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/PayPalMapper.php

```diff
@@ -119,14 +131,24 @@ protected function getDataSubArray(ComputopPayPalPaymentTransfer $computopPayPal
         $dataSubArray[ComputopApiConfig::TRANS_ID] = $computopPayPalPaymentTransfer->getTransId();
         $dataSubArray[ComputopApiConfig::AMOUNT] = $computopPayPalPaymentTransfer->getAmount();
         $dataSubArray[ComputopApiConfig::CURRENCY] = $computopPayPalPaymentTransfer->getCurrency();
-        $dataSubArray[ComputopApiConfig::URL_SUCCESS] = $computopPayPalPaymentTransfer->getUrlSuccess();
-        $dataSubArray[ComputopApiConfig::URL_NOTIFY] = $computopPayPalPaymentTransfer->getUrlNotify();
-        $dataSubArray[ComputopApiConfig::URL_FAILURE] = $computopPayPalPaymentTransfer->getUrlFailure();
         $dataSubArray[ComputopApiConfig::CAPTURE] = $computopPayPalPaymentTransfer->getCapture();
-        $dataSubArray[ComputopApiConfig::RESPONSE] = $computopPayPalPaymentTransfer->getResponse();
-        $dataSubArray[ComputopApiConfig::MAC] = $computopPayPalPaymentTransfer->getMac();
         $dataSubArray[ComputopApiConfig::TX_TYPE] = $computopPayPalPaymentTransfer->getTxType();
         $dataSubArray[ComputopApiConfig::ORDER_DESC] = $computopPayPalPaymentTransfer->getOrderDesc();
+        foreach ($computopPayPalPaymentTransfer->getOrderDescriptions() as $key => $orderDesc) {
+            $dataSubArray[ComputopApiConfig::ORDER_DESC . ($key + 2)] = $orderDesc;
```

Comment:
What is the logic behind `($key + 2)`? Maybe it will be useful to create a separate method with descriptive naming because it's hard to understand why do you need it.

Reply:
Moved to protected function and added description.